bedrock/firefox/templates/firefox/accounts-2018.html

```diff
@@ -0,0 +1,234 @@
+
+    <ul class="mobile-download-buttons mobile-notes">
+      <li class="android">
+        {{ google_play_button(href='https://play.google.com/store/apps/details?id=org.mozilla.testpilot.notes') }}
+      </li>
+    </ul>
+  {% endcall %}
+
+</div>
+
+<aside class="fxa-benefits mzp-u-modal-content">
+  <p class="mobile-title mobile-pocket">{{ _('Scan the QR Code to download the Pocket App on your mobile device.') }}</p>
+  <p class="mobile-title mobile-lockbox">{{ _('Scan the QR Code to download the Lockbox App on your Apple device.') }}</p>
+  <p class="mobile-title mobile-notes">{{ _('Scan the QR Code to download the Notes App on your Android device.') }}</p>
+  <p id="fx-qr-title" class="mobile-title mobile-fx">{{ _('Scan the QR Code to download the Firefox App on your mobile device.') }}</p>
+  <div class="mzp-l-content">
+    <div id="qr-code"></div>
+  </div>
+
+  <section>
+    <div id="fx-app-buttons" class="show-fxa-supported-signed-in">
+        <ul class="mobile-download-buttons">
+          <li>
+            {{ google_play_button(id=android_id, extra_data_attributes={'download-location': button_location}) }}
+          </li>
+          <li>
+            <a href="{{ firefox_ios_url(app_store_campaign) }}" data-link-type="download" data-download-os="iOS" data-download-location="primary cta" id="{{ ios_id }}">
+              <img src="{{ l10n_img('firefox/ios/btn-app-store.svg') }}" alt="{{ _('Download on the App Store') }}" width="152" height="45">
+            </a>
+          </li>
+        </ul>
+      </div>
+  </section>
+
+  <section>
+    <ul class="mobile-download-buttons mobile-pocket">
+      <li class="android">
+        {{ google_play_button(href='https://app.adjust.com/h5iycst') }}
+      </li>
+      <li class="ios">
+        <a href="https://app.adjust.com/h5iycst" data-link-type="download" data-download-os="iOS">
+          <img src="{{ l10n_img('firefox/ios/btn-app-store.svg') }}" alt="{{ _('Download on the App Store') }}" width="152" height="45">
+        </a>
+      </li>
+    </ul>
+  </section>
+
+  <section>
+    <ul class="mobile-download-buttons mobile-lockbox">
+      <li class="ios">
+        <a href="https://app.adjust.com/kq8zhhz" data-link-type="download" data-download-os="iOS">
+          <img src="{{ l10n_img('firefox/ios/btn-app-store.svg') }}" alt="{{ _('Download on the App Store') }}" width="152" height="45">
+        </a>
+      </li>
+    </ul>
+  </section>
+
+  <section>
+    <ul class="mobile-download-buttons mobile-notes">
+      <li class="android">
+        {{ google_play_button(href='https://play.google.com/store/apps/details?id=org.mozilla.testpilot.notes') }}
+      </li>
+    </ul>
+  </section>
+</aside>
+
+<div id="bottom-banner">
+  <div class="mzp-l-content">
+    <div class="banner-title">
+      <p>{{ _('You deserve peace of mind everywhere.') }}</p>
+    </div>
+  </div>
+  <div id="bottom-cta">
```

Comment:
The content breaks out of the standard page container here because this `bottom-cta` div is not a child of the `mzp-l-conent` wrapper above

![screenshot_2018-11-29 get a firefox account keep your data private safe and synced](https://user-images.githubusercontent.com/400117/49219800-47e70300-f3cc-11e8-901a-37510018eb38.png)

Reply:
Going to wrap `<div id="bottom-cta">` in it's own `mzp-l-content` because I would like them to be side by side without more custom css. Is this bad practice? (@stephaniehobson)

Reply:
I don't think it's bad practice but I'm not sure it will fix the problem :/

Both the banner title and the cta need to be inside an element that has a max-width set on it. `mzp-l-content` does have a max width but if the two are size by side instead of in the same one I think it will end up putting the containers side by side but still too wide.

I don't see a way to solve this bug without adding another wrapper in the HTML and moving the grid info on to that.

```
<div id="bottom-banner">
    <div class="mzp-l-content"><!-- this will keep stuff from getting too wide -->
        <div class="bottom-grid"> <!-- move display:grid and other grid properties here-->
            <div class="banner-title">
               ...
            </div>
            <div id="bottom-cta">
              ...
            </div>
        </div>
    </div>
</div>
```

Reply:
Not the only solution of course ;)